main/coursecopy/recycle_course.php

```diff
@@ -55,14 +59,42 @@
         $course = $cb->build();
     }
     $recycle_type = '';
+    $fullDelete = 0;
+    $courseCodeConfirmation ='';
+    if (isset($_POST['course_code_confirmation']))
+        {
+            $courseCodeConfirmation = $_POST['course_code_confirmation'];
+        }
+
     if (isset($_POST['recycle_option']) && $_POST['recycle_option'] === 'full_backup') {
         $recycle_type = 'full_backup';
+        $fullDelete = 1;
+
     } elseif (isset($_POST['action']) && $_POST['action'] === 'course_select_form') {
         $recycle_type = 'select_items';
     }
+
     $cr = new CourseRecycler($course);
-    $cr->recycle($recycle_type);
-    echo Display::return_message(get_lang('RecycleFinished'), 'confirm');
+
+    if ($recycle_type == 'full_backup')
+        {
+            /* to delete, course code confirmation must be equal that current course code */
+            if ($current_course_code == $courseCodeConfirmation)
```

Comment:
Codacy found an issue: [Line indented incorrectly; expected 8 spaces, found 12](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5721344)

Reply:
Spacing needs to be corrected